pages/Category.vue

```diff
@@ -1,125 +1,192 @@
+                />
+                <path
+                  d="m15.717 6.8922h-0.1786c-0.5371-0.0014-1.0525 0.21024-1.4329 0.58921-0.3804 0.37899-0.5942 0.89367-0.5949 1.4302v0.17859c0.0014 0.53649 0.2159 1.0505 0.5956 1.4295 0.3804 0.3783 0.8958 0.5899 1.4322 0.5885h0.1786c1.1138-0.0021 2.0166-0.9056 2.0173-2.0194v-0.17859c-0.0021-1.1138-0.9042-2.0158-2.0173-2.018v7e-5zm1.4414 2.1965c-7e-4 0.79524-0.6461 1.4407-1.4414 1.4414h-0.1786c-0.3839 0.0021-0.753-0.1491-1.0251-0.4191-0.2728-0.27065-0.4261-0.63839-0.4268-1.0223v-0.17859c0.0014-0.79594 0.6462-1.4407 1.4421-1.4421h0.1786c0.3839-0.00211 0.753 0.14906 1.0259 0.42046 0.2721 0.27071 0.4253 0.63915 0.4253 1.023v0.17719z"
+                />
+                <path
+                  d="m9.0959 13.786h-0.19266c-1.1109 5e-3 -2.0095 0.9064-2.0117 2.0173v0.1786c0.00211 1.1116 0.9007 2.0124 2.0117 2.018h0.19266c1.111-0.0057 2.0096-0.9064 2.0117-2.018v-0.1786c-0.0021-1.1109-0.9007-2.0124-2.0117-2.0173zm1.4421 2.1966l-7e-4 -7e-4c-7e-4 0.7959-0.64612 1.4407-1.4414 1.4421h-0.19266c-0.79524-0.0014-1.4407-0.6462-1.4414-1.4421v-0.1786c7e-4 -0.7953 0.64616-1.4407 1.4414-1.4414h0.19266c0.79524 7e-4 1.4407 0.6461 1.4414 1.4414l7e-4 0.1793z"
+                />
+              </g>
+            </svg>
+          </SfIcon>
+          Filters
+        </SfButton>
+        <div class="navbar__sort desktop-only">
+          <span class="navbar__label">Sort by:</span>
         </div>
-      </div>
-    </header>
-    <div class="container pb60">
-      <div class="row m0 pt15">
-        <div class="col-md-3 start-xs category-filters">
-          <sidebar
-            :filters="getAvailableFilters"
-            @changeFilter="changeFilter"
-          />
+        <div class="navbar__counter">
+          <span class="navbar__label desktop-only">
+            Products found:
+          </span>
+          <strong class="desktop-only">{{ getCategoryProductsTotal }}</strong>
+          <span class="navbar__label mobile-only">
+            {{ $t("{count} items", { count: getCategoryProductsTotal }) }}
+          </span>
         </div>
-        <div v-show="mobileFilters" class="col-md-3 start-xs mobile-filters">
-          <div class="close-container absolute w-100">
-            <i class="material-icons p15 close cl-accent" @click="closeFilters"
-              >close</i
-            >
-          </div>
-          <sidebar
-            class="mobile-filters-body"
-            :filters="getAvailableFilters"
-            @changeFilter="changeFilter"
-          />
-          <div class="relative pb20 pt15">
-            <div class="brdr-top-1 brdr-cl-primary absolute divider w-100" />
-          </div>
-          <button-full class="mb20 btn__filter" @click.native="closeFilters">
-            {{ $t("Filter") }}
-          </button-full>
+        <div class="navbar__view desktop-only">
+          <span>View </span>
+          <SfIcon class="navbar__view-icon" size="10px">
+            <svg viewBox="0 0 10 10">
```

Comment:
In my opinion, we should create a separate directory with icons components and import it there. Placing SVG code here makes this file to hard to read.

Reply:
Done.